### PR TITLE
[FIX] 22557: Marking Questions in 5.1

### PR DESCRIPTION
**Pull request by chfsx**
Hi @bheyser 
Currently marked/flagged questions are only displayed in the Dashboard which is a bit confusing. the following PR displays the stars also in the left list of questions

### CODE REVIEW

File: Modules/Test/classes/class.ilTestQuestionSideListGUI.php [hunk 1]
```diff
@@ -190,17 +190,19 @@ private function renderList()
...
-				$tpl->setCurrentBlock('linked_entry');
-				$tpl->setVariable('HREF', $this->buildLink($row['sequence']));
-				$tpl->setVariable('NEXTCMD', ilTestPlayerCommands::SHOW_QUESTION);
-				$tpl->setVariable('NEXTSEQ', $row['sequence']);
-				$tpl->setVariable('CLASS', $class);
-				$tpl->setVariable('ITEM', $title);
-				$tpl->setVariable("DESCRIPTION", $description);
-				$tpl->parseCurrentBlock();
-			/*
+			$tpl->setCurrentBlock('linked_entry');
```
Comment (chfsx) [inline Modules/Test/classes/class.ilTestQuestionSideListGUI.php:193]: Line 5-12 is only code format

**Comment by Amstutz** [comment]
Technical Board: This seems to be quite old, @bheyser please give some feedback concerning the state of this.

File: Modules/Test/classes/class.ilTestQuestionSideListGUI.php [hunk 2]
```diff
@@ -190,17 +190,19 @@ private function renderList()
...
+			$tpl->setVariable('HREF', $this->buildLink($row['sequence']));
+			$tpl->setVariable('NEXTCMD', ilTestPlayerCommands::SHOW_QUESTION);
+			$tpl->setVariable('NEXTSEQ', $row['sequence']);
+			$tpl->setVariable('CLASS', $class);
+			$tpl->setVariable('ITEM', $title);
+			$tpl->setVariable("DESCRIPTION", $description);
+
+			if ($row['marked']) {
```
Comment (chfsx) [inline Modules/Test/classes/class.ilTestQuestionSideListGUI.php:201]: This is the relevant part

**Comment by bheyser** [comment]
I can remember a heavy discussion on the SIG EA meeting that took place in Dortmund about exactly this topic. It was during the conceptional discussions about Submission and Navigation (first implementation).

As far as I remember the final decision was too NOT add the marked questions action/status star icon to the questions list displayed at the side of the working form.

Therefore I would like to discuss this topic on the JourFixe.

**Comment by matthiaskunkel** [comment]
Jour Fixe, 05 FEB 2018: Please open a Mantis bug report for this problem. @Björn: please contact the SIG T&A if this bugfix is accepted (to prevent future discussions).

**Comment by chfsx** [comment]
A ticket has been reported: https://www.ilias.de/mantis/view.php?id=22557

**Comment by matthiaskunkel** [comment]
Jour Fixe, 26 FEB 2018: Due to conceptual questions we ask to close this PR and to create a new one in coordination with Björn Heyser.

**Pull request closed without merging**